marketpulse/main/templates/fxosprice_view.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2 class="title">Firefox OS devices prices</h2>
+  <h3 class="title">your contributions</h3>
```

Comment:
I would prefer the text here to be ``Your contributions``. Also it makes sense for me these ``<h2>`` and ``<h3>`` elements to be shown more separate, or <h2> to be more emphasized. I also tried locally with ``text-align: center`` and it doesn't look bad.